src/fides/api/models/detection_discovery.py

```diff
@@ -348,6 +338,41 @@ def mark_as_addition(
                 parent_resource.add_child_diff_status(DiffStatus.ADDITION)
 
 
+class MonitorExecution(Base):
+    """
+    Monitor execution record used for data detection and discovery.
+
+    Each monitor execution references `MonitorConfig`, which provide it with underlying
+    configuration details used in connecting to the external data store.
+    """
+
+    id = Column(String, primary_key=True)
+    monitor_config_key = Column(
+        String,
+        ForeignKey(MonitorConfig.key),
+        nullable=False,
+        index=True,
+    )
+    status = Column(String, nullable=True)
+    started = Column(
+        DateTime(timezone=True), nullable=True, default=datetime.now(timezone.utc)
+    )
```

Comment:
nit: we _could_ just use `created_at` as our de-facto "started" timestamp i think? we do that on some other models, and reduces some redundancy. but it's OK if we wanna keep it as a separate property/attribute

Reply:
We could, I left it this way just to make the serialization to the pydantic model easier but I can make it mapped manually if we prefer to just rely on one column. Or I could remove `created_at`?

Reply:
don't think we should remove `created_at` because it's basically on all of our tables. the pydantic model serialization makes sense, though another option for that is to provide an `alias` on the pydantic field so that e.g. in the API response it come through as `started` even though underneath it's being referred to as `created_at`.

that being said, i think it's fine to leave as-is 👍